pkg/config/config.go

```diff
@@ -177,12 +177,16 @@ type Config struct {
 			SecretKey SecureString `mapstructure:"secret_key" validate:"required"`
 		} `mapstructure:"encrypt"`
 		API struct {
+			// Endpoint is the endpoint to used for authorization operations
 			Endpoint           string        `mapstructure:"endpoint"`
 			Token              SecureString  `mapstructure:"token"`
 			SupportsInvites    bool          `mapstructure:"supports_invites"`
 			HealthCheckTimeout time.Duration `mapstructure:"health_check_timeout"`
 			SkipHealthCheck    bool          `mapstructure:"skip_health_check"`
 		} `mapstructure:"api"`
+		AuthenticationAPI struct {
+			Endpoint string `mapstructure:"endpoint"`
```

Comment:
JUST AN IDEA:

```suggestion
			// Endpoint for authentication operations
			Endpoint string `mapstructure:"endpoint"`
```
to make the difference clearer.